### DISCUSSION TITLE
特定の行と列のキーが反応しない

### DISCUSSION THREAD

**Discussion by moko-poi** [Yowkees/keyball#656] [Q&A]
お世話になっております。
keyball61にて、画像で赤枠で示した列のキーが全て反応しません。
原因となりうる部分やテスト方法をご教示いただけますでしょうか？
お手隙の際によろしくお願い致します。
![IMG_3997](https://github.com/user-attachments/assets/9e32bd38-4198-43af-8cdc-99ec0d06b2cd)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。状況了解しました。

まず入力できないキーですが、下記画像の緑丸で示したダイオードがしっかり付いていないとご指摘の１列のキーが動作しません。
このダイオードの極性確認と再半田付けをお願いします。

![Keyball61_rightfix](https://github.com/user-attachments/assets/89d8e6c3-9e6c-42eb-b0ee-b3ddc419c9c9)

半田付けに問題がないようでしたら、上画像のオレンジで示した２点が導通しているかご確認ください。"COL1"と表示がある端子です。

お手数ですがご確認よろしくお願いいたします。

**Reply by moko-poi** [reply]
ご返信ありがとうございます！
オレンジの箇所を半田付けし直したところ解決しました！
ありがとうございました🙇

**Comment by Yowkees** [comment]
早速のご対応ありがとうございます。無事に直ったようで安心しました。
ぜひ末永くご愛用いただければ幸いです。
